v1/features/1.0/index.php

```diff
@@ -0,0 +1,119 @@
+<?php
+
+/**
+ * Query petition / voting / features requsts for ClassicPress
+ * This API serves as a CORS compliant proxy service to fetch this data on behalf of ClassicPress installations
+ *
+ * $mode could be useful to parse incoming request variables and change what happens, once a more developed request
```

Comment:
> $mode could be useful to parse incoming request variables and change what happens, once a more developed request router is created

Let's remove this until and unless we need it.

For what it's worth, I'm not sure we actually want a more sophisticated request routing mechanism here. This code should be very minimal so that it is easy to audit, solid, and fast.